lib/index.js

```diff
@@ -0,0 +1,57 @@
+ *                       any "includes" files specified in the frontmatter,
+ *                       "prelude" content if specified (see below), and any
+ *                       "scenario" transformations
+ * - attrs {object} - an object representation of the metadata declared in the
+ *                    test's YAML-formatted "frontmatter" section
+ * - copyright {string} - the licensing information included within the test
+ *                        (if any)
+ * - scenario {string} - name describing how the source file was interpreted to
+ *                       create the test
+ * - async {boolean} - *deprecated*; an unreliable indicator of whether the
+ *                     test describes asynchronous behavior; this information
+ *                     is consistently available in the `async` metadata flag
+ * - isATest {boolean} - *deprecated*; an unreliable indicator of whether the
+ *                       object describes a test; all emitted values describe
+ *                       tests
+ * - strictMode {boolean} - *deprecated*; indicator of whether a global "use
+ *                          strict" directive was inserted into the contents;
+ *                          this does *not* definitely describe the strictness
+ *                          of the code
+ *
+ * @param {string} test262Dir - filesystem path to a directory containing
+ *                              Test262
+ * @param {object} [options]
+ * @param {string} [options.includesDir] - directory from which to load
+ *                                         "includes" files (defaults to the
+ *                                         appropriate subdirectory of the
+ *                                         provided `test262Dir`
+ * @param {Array<string>} [options.paths] - file system paths refining the set
+ *                                          of tests that should be produced;
+ *                                          only tests whose source file
+ *                                          matches one of these values (in the
+ *                                          case of file paths) or is contained
+ *                                          by one of these paths (in the case
+ *                                          of directory paths) will be
+ *                                          created; all paths are interpreted
+ *                                          relative to the root of the
+ *                                          provided `test262Dir`
+ * @param {string} [options.prelude] - string contents to inject into each
+ *                                     test that does not carry the `raw`
+ *                                     metadata flag; defaults to the empty
+ *                                     string (e.g. no injection)
+ *
+ * @returns {object} readable stream
+ */
+exports.streamTests = function(test262Dir, options) {
+  return new TestStream(test262Dir, options);
+};
```

Comment:
Export the class `TestStream` itself (which also means this file is unnecessary, but the docs can move as well)

Reply:
I've pushed up a commit to expose `TestStream`. To your point about `index.js`: I believe that file is still necessary, if only to provide a namespace to expose the `TestStream` class (since the `test-stream.js` module exports the function directly).